src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/LoginPersonaFisicaPagoPA.java

```diff
@@ -152,6 +159,72 @@ public void loginConDestinatario(String datipersonaFisica) {
         confermaDatiSpidPFPage.selezionaConfermaButton();
         headerPFSection.waitUrlToken();
     }
+    @When("Login con persona fisica")
```

Comment:
why you do this method, login pf already exists

Reply:
take a token for login with another pf and refactorize method passing him token in parameter

Reply:
there is no token for colombo. only token for pf is user Cesare